bin/polly.py

```diff
@@ -452,7 +457,17 @@ def PositiveInt(string):
     timer.stop()
   if args.pack:
     timer.start('Pack')
-    detail.pack_command.run(args.config, logging, cpack_generator)
+
+    if args.cpack:
+      cpack_bin = args.cpack
+    else:
+      cpack_bin = 'cpack'
+
+    if os.path.isabs(cpack_bin):
+      if not os.path.exists(cpack_bin):
+        sys.exit("CPack binary not found: {}".format(cmake_bin))
+
+    detail.pack_command.run(args.config, logging, cpack_generator, cpack_bin, cmake_bin)
```

Comment:
Apologies on that, I missed that file and was in the middle of teaching the script about ctest_bin as well as I was hitting that case as I use this script.